v20.2/alter-user.md

```diff
@@ -32,14 +32,78 @@ table td:first-child {
 
 Parameter | Description
 ----------|-------------
-`name` | The name of the user whose password you want to create or add.
-`password` | Let the user [authenticate their access to a secure cluster](authentication.html#client-authentication) using this new password. Passwords should be entered as a [string literal](sql-constants.html#string-literals). For compatibility with PostgreSQL, a password can also be entered as an [identifier](#change-password-using-an-identifier). <br><br>To prevent a user from using [password authentication](authentication.html#client-authentication) and to mandate [certificate-based client authentication](authentication.html#client-authentication), [set the password as `NULL`](#prevent-a-user-from-using-password-authentication).
+`name` | The name of the user whose role options you want to alter.
```

Comment:
Is this table the same as in the other page?  Might (?) be worth doing an include eventually if there becomes a 3rd copy somewhere.